Framework/Source/Utils/Gui.h

```diff
@@ -92,6 +92,14 @@ namespace Falcor
         */
         bool onKeyboardEvent(const KeyboardEvent& event);
 
+        /** Display image within imgui
+            \param[in] label. Name for id for item.
+            \param[in] pTex. Pointer to texture resource to draw in imgui
+            \param[in] size. Size in pixels of the image to draw
+            \param[in] sameLine Optional. If set to true, the widget will appear on the same line as the previous widget
+        */
+        void addImage(const char label[], const Texture::SharedPtr& pTexSrv, const glm::vec2& size = {128.0f, 128.0f}, bool sameLine = false);
```

Comment:
The comment doesn't match the actual names. It's also not a `TexSrv`, it's just a texture